### PR TITLE
Document PingOpts

### PR DESCRIPTION
**Pull request by newhoggy**
# Description

Document `PingOpts` data type.

# Checklist

- Branch
    - [ ] Commit sequence broadly makes sense
    - [ ] Commits have useful messages
    - [ ] New tests are added if needed and existing tests are updated
    - [ ] If this branch changes Consensus and has any consequences for downstream repositories or end users, said changes must be documented in [`interface-CHANGELOG.md`](../ouroboros-consensus/docs/interface-CHANGELOG.md)
    - [ ] If this branch changes Network and has any consequences for downstream repositories or end users, said changes must be documented in [`interface-CHANGELOG.md`](../docs/interface-CHANGELOG.md)
    - [ ] If serialization changes, user-facing consequences (e.g. replay from genesis) are confirmed to be intentional.
- Pull Request
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description at least containing the following information:
      - What does this PR change?
      - Why these changes were needed?
      - How does this affect downstream repositories and/or end-users?
      - Which ticket does this PR close (if any)? If it does, is it [linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)?
    - [ ] Reviewer requested

### CODE REVIEW

**Review by coot — APPROVED** [review]
LGTM

File: cardano-ping/src/Cardano/Network/Ping.hs [hunk 1]
```diff
@@ -50,12 +50,19 @@ import qualified System.IO as IO
 
 data PingOpts = PingOpts
   { pingOptsCount    :: Word32
+    -- ^ Number of messages to send to the server
   , pingOptsHost     :: (Maybe String)
+    -- ^ The host to connect to
   , pingOptsUnixSock :: (Maybe String)
```
Comment (Jimbo4350) [inline cardano-ping/src/Cardano/Network/Ping.hs:56]: Redundant brackets

Comment (newhoggy) [reply cardano-ping/src/Cardano/Network/Ping.hs]: Fixed!

**Pull request merged**